izanami-frontend/src/components/FeatureTable.tsx

```diff
@@ -491,18 +491,27 @@ function OperationTagForm(props: {
   } else if (tagsQuery.error) {
     return <div className="error">Failed to load tags</div>;
   } else {
+    const selectedTags = selectedRows.flatMap((row) => row.tags);
+
+    const nonRepeatingTags =
```

Comment:
Using a Set is a simpler and more readable way of deduplicated elements [See here for more details](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Set)